packages/theme/src/typography.ts

```diff
@@ -37,54 +48,66 @@ export type NativeTypographyStyles = Record<TypographyStyle, NativeTypographySty
 
 // we need unit-less typography base because RN is unit-less, we can easily add units later
 // for web we need string instead of object because styled-components syntax
-export const typographyStylesBase: Record<TypographyStyle, TypographyStyleDefinition> = {
-    titleLarge: {
-        fontSize: 48,
-        lineHeight: 53,
-        fontWeight: fontWeights.medium,
-        letterSpacing: 0.4,
-    },
-    titleMedium: {
-        fontSize: 34,
-        lineHeight: 37,
-        fontWeight: fontWeights.medium,
-        letterSpacing: -1.4,
-    },
-    titleSmall: {
-        fontSize: 22,
-        lineHeight: 32,
-        fontWeight: fontWeights.medium,
-        letterSpacing: -0.3,
-    },
-    highlight: {
-        fontSize: 16,
-        lineHeight: 24,
-        fontWeight: fontWeights.semiBold,
-        letterSpacing: -0.4,
-    },
-    body: {
-        fontSize: 16,
-        lineHeight: 24,
-        fontWeight: fontWeights.medium,
-        letterSpacing: -0.4,
-    },
-    callout: {
-        fontSize: 14,
-        lineHeight: 20,
-        fontWeight: fontWeights.semiBold,
-        letterSpacing: -0.3,
-    },
-    hint: {
-        fontSize: 14,
-        lineHeight: 20,
-        fontWeight: fontWeights.medium,
-        letterSpacing: -0.3,
-    },
-    label: {
-        fontSize: 12,
-        lineHeight: 18,
-        fontWeight: fontWeights.medium,
-        letterSpacing: -0.1,
+export const nativeTypographyStylesBase: Record<NativeTypographyStyle, TypographyStyleDefinition> =
+    {
+        titleLarge: {
+            fontSize: 48,
+            lineHeight: 53,
+            fontWeight: fontWeights.medium,
+            letterSpacing: 0.4,
+        },
+        titleMedium: {
+            fontSize: 34,
+            lineHeight: 37,
+            fontWeight: fontWeights.medium,
+            letterSpacing: -1.4,
+        },
+        titleSmall: {
+            fontSize: 22,
+            lineHeight: 32,
+            fontWeight: fontWeights.medium,
+            letterSpacing: -0.3,
+        },
+        highlight: {
+            fontSize: 16,
+            lineHeight: 24,
+            fontWeight: fontWeights.semiBold,
+            letterSpacing: -0.4,
+        },
+        body: {
+            fontSize: 16,
+            lineHeight: 24,
+            fontWeight: fontWeights.medium,
+            letterSpacing: -0.4,
+        },
+        callout: {
+            fontSize: 14,
+            lineHeight: 20,
+            fontWeight: fontWeights.semiBold,
+            letterSpacing: -0.3,
+        },
+        hint: {
+            fontSize: 14,
+            lineHeight: 20,
+            fontWeight: fontWeights.medium,
+            letterSpacing: -0.3,
+        },
+        label: {
+            fontSize: 12,
+            lineHeight: 18,
+            fontWeight: fontWeights.medium,
+            letterSpacing: -0.1,
+        },
+    };
+// we need unit-less typography base because RN is unit-less, we can easily add units later
```

Comment:
This comment is a copy and doesn't make sense here.

Reply:
Done